portability-core/src/main/java/org/dataportabilityproject/job/CrypterImpl.java

```diff
@@ -35,14 +36,19 @@ public String encrypt(String data) {
       byte[] encrypted = cipher.doFinal(data.getBytes(Charsets.UTF_8));
       return BaseEncoding.base64Url().encode(encrypted);
     } catch (BadPaddingException e) {
+      logger.error("BadPaddingException for data, length: {}", data.length(), e);
```

Comment:
Out of curiosity, since we don't do anything different for these, (aside from the name of the exception), can we put them all in one catch?